README.md

```diff
@@ -16,7 +16,7 @@
 ## Overview
 
 This is the **future** command line tool for Mbed OS. It provides the ability to detect Mbed Enabled devices connected
-by USB, checkout Mbed projects and perform builds amongst other operations.
+by USB, deploy Mbed projects and perform builds amongst other operations.
```

Comment:
We need to change this wording to something that makes sense. As a reader of this I'm not sure where or how we're deploying the mbed project, or what deploying even means in the context of an mbed project.